### PR TITLE
add ordhook docs

### PR DESCRIPTION
**Pull request by ryanwaits**
Adding Ordhook documentation to sidebar

### CODE REVIEW

File: docs/ordhook/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: Getting Started
```
Comment (sabbyanandan) [inline docs/ordhook/getting-started.md:2]: A meta-comment:

It might be a good idea to update the README with these instructions and then pull the README into the docs here. You can look at this approach in action for Clarinet for reference.

File: docs/ordhook/how-to-guides/how-to-explore-ordinal-activities.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
+---
+title: Explore Ordinal activities in your terminal
+---
+
+Ordhook can be used to extract ordinal activities from the Bitcoin chain and stream these activities to the indexer. This guide helps you with the steps to explore ordinal activities and post these activities to the indexer.
```
Comment (sabbyanandan) [inline docs/ordhook/how-to-guides/how-to-explore-ordinal-activities.md:5]: The "indexer" part, while it is understandable by us, I doubt the general audience would. 

We could either describe the indexer beforehand or explain downstream step more generally (e.g., building your database)

File: docs/ordhook/how-to-guides/how-to-explore-ordinal-activities.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+```
+Inscription 6fb976ab49dcec017f1e201e84395983204ae1a7c2abf7ced0a85d692e442799i0 revealed at block #767430 (inscription_number 0, ordinal_number 1252201400444387)
+        → Transferred in transaction 0x2c8a11858825ae2056be90c3e49938d271671ac4245b452cd88b1475cbea8971 (block #785391)
+        → Transferred in transaction 0xbc4c30829a9564c0d58e6287195622b53ced54a25711d1b86be7cd3a70ef61ed (block #785396)
+Number of transfers: 2
+```
+
+### Configure ordhook
+
+This section walks you through streaming ordinal activities to an indexer. To post the ordinal activity, you'll need to configure bitcoind. Refer to [Setting up a bitcoin node](https://docs.hiro.so/chainhook/how-to-guides/how-to-run-chainhook-as-a-service-using-bitcoind#setting-up-a-bitcoin-node) to understand the steps to configure Bitcoind.
```
Comment (sabbyanandan) [inline docs/ordhook/how-to-guides/how-to-explore-ordinal-activities.md:37]: Because "bitcoind" is required for Ordhook, it would be good to have explicit instructions in this document or reuse the same MD file/content in both Chainhook and Ordhook.

I say this because, once the developer context switches to Chainhook, the information fidelity might be lost, and there's no direct way to get back to Ordhook other than clicking the "back" buttons.

Comment (sabbyanandan) [reply docs/ordhook/how-to-guides/how-to-explore-ordinal-activities.md:37]: If it is a bigger chunk of work to get it done as part of this PR, the cross-link reference is fine for now, and we can improve the extraction+reuse in a different PR.

Just listing a few suggestions.

File: docs/ordhook/how-to-guides/how-to-use-ordhook-as-a-service-using-bitcoind.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+---
+title: Run Ordhook as a Service using Bitcoind
+---
+
+## Run ordhook as a service using Bitcoind
+
+In this section, you'll learn how to run Ordhook as a service using [Chainhook SDK](https://github.com/hirosystems/chainhook/tree/develop/components/chainhook-sdk) to create a predicate ordinal theory and post events to a server.
```
Comment (sabbyanandan) [inline docs/ordhook/how-to-guides/how-to-use-ordhook-as-a-service-using-bitcoind.md:7]: The "create a predicate ordinal theory" language could be hard to follow. We could consider rephrasing this so our audience can understand it.

File: docs/ordhook/how-to-guides/how-to-use-ordhook-as-a-service-using-bitcoind.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+## Prerequisites
+
+- [Bitcoind configuration](https://docs.hiro.so/chainhook/how-to-guides/how-to-run-chainhook-as-a-service-using-bitcoind#setting-up-a-bitcoin-node)
+- Configure ordhook by following [this](./how-to-explore-ordinal-activities.md#configure-ordhook) section
```
Comment (sabbyanandan) [inline docs/ordhook/how-to-guides/how-to-use-ordhook-as-a-service-using-bitcoind.md:12]:
```suggestion
- Configure Ordhook by following [this](./how-to-explore-ordinal-activities.md#configure-ordhook) section
```

File: docs/ordhook/how-to-guides/how-to-use-ordhook-as-a-service-using-bitcoind.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+## Run ordhook service for streaming blocks
+
+Use the following command to start the ordhook service for streaming and processing new blocks appended to the Bitcoin blockchain.
```
Comment (sabbyanandan) [inline docs/ordhook/how-to-guides/how-to-use-ordhook-as-a-service-using-bitcoind.md:16]:
```suggestion
Use the following command to start the Ordhook service for streaming and processing new blocks appended to the Bitcoin blockchain.
```

File: docs/ordhook/how-to-guides/how-to-use-ordhook-as-a-service-using-bitcoind.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+`ordhook service start --post-to=http://localhost:3000/api/events --config-path=./Ordhook.toml`
+
+When the ordhook service starts, the chainhook service gets initiated in the background to augment the blocks that Bitcoind sends. The bitcoind sends zeromq notifications to Ordhook to retrieve the inscriptions.
```
Comment (sabbyanandan) [inline docs/ordhook/how-to-guides/how-to-use-ordhook-as-a-service-using-bitcoind.md:20]:
```suggestion
When the Ordhook service starts, the chainhook service gets initiated in the background to augment the blocks from Bitcoin. Bitcoind sends ZeroMQ notifications to Ordhook to retrieve the inscriptions.
```

Comment (sabbyanandan) [reply docs/ordhook/how-to-guides/how-to-use-ordhook-as-a-service-using-bitcoind.md:20]: To use Ordhook, we do not need Chainhook, but just a Bitcoin node. At least that is my interpretation. I can stand corrected, though.

@lgalabru: Thoughts?

File: docs/ordhook/how-to-guides/how-to-use-ordhook-as-a-service-using-bitcoind.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+### Add `http-post` endpoints dynamically
+
+To enable dynamically posting endpoints to the server, you can spin up the redis server by enabling the following lines of code in the `ordhook.toml` file.
```
Comment (sabbyanandan) [inline docs/ordhook/how-to-guides/how-to-use-ordhook-as-a-service-using-bitcoind.md:24]:
```suggestion
To enable dynamically posting endpoints to the server, you can spin up the Redis server by enabling the following lines of code in the `ordhook.toml` file.
```

Comment (sabbyanandan) [reply docs/ordhook/how-to-guides/how-to-use-ordhook-as-a-service-using-bitcoind.md:24]: This statement can be explained with why Redis is required to expose RESTful endpoints.

File: docs/ordhook/how-to-guides/how-to-use-ordhook-as-a-service-using-bitcoind.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+database_uri = "redis://localhost:6379/"
+```
+
+## Run ordhook service
+
+Based on the `Ordhook.toml` file configuration, the ordhook service spins up an HTTP API to manage event destinations. Use the following command to start the ordhook service.
+
+`ordhook service start --config-path=./Ordhook.toml`
+
+A comprehensive OpenAPI specification explaining how to interact with this HTTP REST API can be found [here](https://github.com/hirosystems/chainhook/blob/develop/docs/chainhook-openapi.json).
```
Comment (sabbyanandan) [inline docs/ordhook/how-to-guides/how-to-use-ordhook-as-a-service-using-bitcoind.md:38]: This is pointing to the spec file in Chainhook repo, is it expected?

File: docusaurus.config.js [hunk 10]
```diff
@@ -193,6 +193,28 @@ module.exports = {
         },
       },
     ],
+    [
+      'docusaurus-plugin-remote-content',
+      {
+        name: 'remote-docs-ordhook-docs',
+        sourceBaseUrl: 'https://raw.githubusercontent.com/hirosystems/ordhook/develop/docs/',
+        outDir: 'docs/ordhook',
+        documents: ['getting-started.md', 'overview.md'],
```
Comment (sabbyanandan) [inline docusaurus.config.js:202]: Oh wait. You are pulling the remote files!

**Review by sabbyanandan — CHANGES_REQUESTED** [review]
A few subtle change-edits and nits. 

More importantly, you are pulling the remote files correctly, but there are also some local changes to the pulled files, which can be double-checked to be sure.

File: docs/ordhook/how-to-guides/how-to-explore-ordinal-activities.md (hunk 3 above)
Comment (ryanwaits) [reply docs/ordhook/how-to-guides/how-to-explore-ordinal-activities.md:37]: this is good feedback, appreciate it!

File: docusaurus.config.js (hunk 10 above)
Comment (ryanwaits) [reply docusaurus.config.js:202]: ha yeah i started with local files, wasn't sure if they would get pulled in to test out locally but then i figured they would just get it overwritten with any changes from the remote repo

File: docs/ordhook/how-to-guides/how-to-use-ordhook-as-a-service-using-bitcoind.md (hunk 9 above)
Comment (ryanwaits) [reply docs/ordhook/how-to-guides/how-to-use-ordhook-as-a-service-using-bitcoind.md:38]: yeah i believe that is the only reference to the openapi spec, but i could copy/paste that and create a new `ordhook-openapi.json` file to avoid the confusion

File: docs/ordhook/how-to-guides/how-to-use-ordhook-as-a-service-using-bitcoind.md (hunk 7 above)
Comment (lgalabru) [reply docs/ordhook/how-to-guides/how-to-use-ordhook-as-a-service-using-bitcoind.md:20]: You're correct @sabbyanandan!

Comment (lgalabru) [reply docs/ordhook/how-to-guides/how-to-use-ordhook-as-a-service-using-bitcoind.md:20]: Ordhook is embedding `chainhook-sdk`

File: docs/ordhook/getting-started.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+---
+
+Ordhook is a utility designed for interacting with the [Ordinal Theory](https://trustmachines.co/glossary/ordinal-theory) protocol, enabling you to assign distinct identifiers to newly minted satoshis for diverse applications. Follow the steps below to install Ordhook.
```
Comment (lgalabru) [inline docs/ordhook/getting-started.md:5]: `utility`: `tool`? `command line tool`? `developer tool`?

File: docs/ordhook/getting-started.md [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+```
+
+You are now inside the Ordhook directory.
+
+### Install Ordhook
+
+Use the Rust package manager, Cargo, to install Ordhook. Run the following command:
+
+```bash
+cargo ordhook-install
```
[the remark this answers is not in the document]
Comment (ryanwaits) [reply docs/ordhook/getting-started.md:37]: updates here: https://github.com/hirosystems/ordhook/pull/184

File: docs/ordhook/getting-started.md (hunk 11 above)
Comment (ryanwaits) [reply docs/ordhook/getting-started.md:5]: updates here: https://github.com/hirosystems/ordhook/pull/184

**Review by sabbyanandan — CHANGES_REQUESTED** [review]
@ryanwaits: Your assembly of remote docs from https://github.com/hirosystems/ordhook/tree/develop/docs and the related config changes looks good; however, I am confused as to why you would want the `overview.md` and `getting-started.md` files locally also. 

They are at the upstream repo, anyway: https://github.com/hirosystems/ordhook/tree/develop/docs

I think simply pointing to those files where the remote plugin dumped in the `sidebar.js` is all you would need. 

I could be misinterpreting, and I am on standby to be corrected.

**Comment by ryanwaits** [comment]
yeah that's nbd, i deleted them locally and will have them ignored by git moving forward

otherwise i think these updates should be in a good spot to go ahead and publish, lmk if we're good to approve!

**Review by sabbyanandan — APPROVED** [review]
Heck yeah! Looks good now. Thanks, @ryanwaits!
